### PR TITLE
[WIP] Don't waste my time

### PR DESCRIPTION
**Pull request by vtno**

### CODE REVIEW

**Comment by chawisara** [comment]
ok i forgot what i changed la but try comparing this with urs ahahahah

##Consequences
Because of that first product I built, I got to work part-time with my senior who is working on a Rails project for government sector at his company. It was a pretty big project that I actually learned the best practices of how to create a “good” Rails app. After this project, I applied to one of the coolest tech company in town and they hired me.

If my goal was just to learn Rails, I would still be reading some tutorial off of the internet and be nowhere near I am today. It would take a lot more time and I could miss my chance of applying at my desired company.

#Don’t plan too much
##You’ll just end up doing nothing
Planning is an essential part in making a great decision and achieve more results in less time. However, every great result is not going to be achieved if you don’t execute. Have you ever seen those people who have a long-ass list of New Year Resolution they gave up?

File: dont_waste_your_time.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ It was crappy but it worked. I’ve built my first product.
...
 
 If my goal was just to learn Rails, I would still be reading some tutorial off of the internet and be nowhere near I am today. It would take a lot more time and I could miss my chance of applying at my desired company.
 
 #Don’t plan too much
-##You’ll just end up doing nothing
-Planning is an essential part in making a great decision and achieve more result in less time. However, every great result is not going to be achieve if you don’t execute. Have you ever see those people who have a long-ass list of New Year Resolution they gave up? What ever it is you plan to
+######You’ll just end up doing nothing
+Planning is an essential part in making a great decision and achieve more result in less time. However, every great result is not going to be achieve if you don’t execute. Having a master plan that wil pave the way for your future means nothing if you don't start executing it. Have you ever talked to someone who always going through their plans...
```
Comment (vtno) [inline dont_waste_your_time.md]: @belza09 You could comment like this. It's awesome

Comment (chawisara) [inline dont_waste_your_time.md]: OH SHITTTT so coolll never tried it before

File: dont_waste_your_time.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ It was crappy but it worked. I’ve built my first product.
 [My First Rails App](https://github.com/vtno/clothstore)
 
 ######Consequences
-Because of that first product I built, I got to work part-time with my senior who is working on a Rails project for government sector at his company. It is a pretty big project that I actually learn the best practices of how to create a “good” Rails app. After this project, I apply at one of the coolest tech company in town and they hired me.
+Because of that first product I built, I got to work part-time with my senior who is working on a Rails project for government sector at his company. It was a pretty big project that I actually learn the best practices of how to create a “good” Rails app. After this project, I applied to one of the coolest tech company in town and they hired me.
```
Comment (vtno) [inline dont_waste_your_time.md:38]: @belza09  I fixed grammar right here. Please review them lol.

**Comment by vtno** [comment]
@belza09 I just added some more stuff. Please review them and I'll get them on medium
 😃

File: dont_waste_your_time.md [hunk 3]
```diff
@@ -41,4 +41,11 @@ If my goal was just to learn Rails, I would still be reading some tutorial off o
 
 #Don’t plan too much
 ######You’ll just end up doing nothing
-Planning is an essential part in making a great decision and achieve more result in less time. However, every great result is not going to be achieve if you don’t execute. Having a master plan that wil pave the way for your future means nothing if you don't start executing it. Have you ever talked to someone who always going through their plans...
+Planning is an essential part in making a great decision and achieve more result in less time. During my college, I worked with different kinds of people. People from engineering school tends to plan for everything and I really mean __every freaking thing possible__. Even the smallest detail, they won't let it slip and might even spent a couple hours on finding fixed for that small issue.
```
Comment (chawisara) [inline dont_waste_your_time.md:44]: more result >> more results

File: dont_waste_your_time.md [hunk 4]
```diff
@@ -41,4 +41,11 @@ If my goal was just to learn Rails, I would still be reading some tutorial off o
...
+
+I love working meticulously however sometimes you just need to go with what you've got hope for the best. You just can't plan for every scenario. My current mindset on planning is not to go into much detail. I just set my goal, break it down and set which step I have to take. Plan for the worst and just go with it. I don't waste my time thinking of how many way my plan could to fail. Sometimes you just need to go for it. No one will wait for you to finish your plan. Time wasted in planning could make you miss your opportunity or your plan might become obsolete because you're taking too long.
```
Comment (chawisara) [inline dont_waste_your_time.md:46]: how many way >> ways
my plan could to fail >> delete 'to'

File: dont_waste_your_time.md [hunk 5]
```diff
@@ -41,4 +41,11 @@ If my goal was just to learn Rails, I would still be reading some tutorial off o
...
+
+It's better to take the opportunity in front of you than come up with a plan on what you're going to do with it, only to find out that you just let it slipped.
```
Comment (chawisara) [inline dont_waste_your_time.md:48]: let it slipped >> slip

File: dont_waste_your_time.md (hunk 2 above)
Comment (chawisara) [inline dont_waste_your_time.md:38]: i actually learn >> learned naa ja dee gwa na

**Pull request merged**